### PR TITLE
Noel c9 patch4

### PR DESCRIPTION
**Pull request by NoelC9**

### CODE REVIEW

**Review by dgiri12 — DISMISSED** [review]

**Review by sbowma15 — APPROVED** [review]
good call

**Review by sbowma15 — APPROVED** [review]
Noice

**Review by sbowma15 — APPROVED** [review]
Looks great.  Is 185-187 unnecessary ?, good thinking with line 193. Simple and efficient

File: src/main/java/com/cs3250p1/project1/BarChartEx.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
+package com.cs3250p1.project1;
```
Comment (dgiri12) [inline src/main/java/com/cs3250p1/project1/BarChartEx.java:1]: Author is @NoelC9

File: src/main/java/com/cs3250p1/project1/Maindif.java [hunk 2]
```diff
@@ -0,0 +1,323 @@
...
+    });
+
+    gui.getButton(btnCrudOpenCustomerOrder).addActionListener(new ActionListener() {
+        @Override
+        public void actionPerformed(ActionEvent e) {
+            ArrayList<String> lines = gui.getStringFromFileDialog(); //lets user choose a file,
+            // then loads it's string as a list onto the variable
+            //parse that string and store it into a structure
+
+            for(int i = 0; i < lines.size(); i++){
```
Comment (dgiri12) [inline src/main/java/com/cs3250p1/project1/Maindif.java:273]: Author @NoelC9  for split function usage in action listener

File: src/main/java/com/cs3250p1/project1/OrderDAO.java [hunk 3]
```diff
@@ -1,5 +1,6 @@
 package com.cs3250p1.project1;
```
Comment (dgiri12) [inline src/main/java/com/cs3250p1/project1/OrderDAO.java:1]: author and optimizer @NoelC9

File: src/main/java/com/cs3250p1/project1/SalesOrderdif.java [hunk 4]
```diff
@@ -0,0 +1,89 @@
+package com.cs3250p1.project1;
+import java.util.*;
+
```
Comment (dgiri12) [inline src/main/java/com/cs3250p1/project1/SalesOrderdif.java:3]: this is useless file, just to resolve conflict that GitHub was complaining about

**Review by dgiri12 — COMMENTED** [review]
Browsed through the files, put credits in right places

**Pull request merged**